### ISSUE TITLE
Change struct instantiation syntax

### ISSUE THREAD

**Issue by Champii** [Champii/Rock#26]
Current syntax is not readable

```haskell
let inst = Foo
  foo: 42
  bar: "test"
```

We should at least have

```haskell
let inst = 
  Foo
    foo: 42
    bar: "test"
```

and allow for one-liners

```haskell
let inst = Foo foo: 42, bar: "test"
```

**Comment by Champii** [comment]
Multiline struct instantitation Fixed in #30
Need one liner

**Comment by Champii** [comment]
The new parser does not implement one-liners or even the new multiline format.
This needs to be fixed as the README on master uses this new format.
